### DISCUSSION TITLE
Non-Windows Certificate Enrollment Community version

### DISCUSSION THREAD

**Discussion by BenjaminGC** [Keyfactor/ejbca-ce#371] [Q&A]
Hello everyone, I just finished building my first tier 2 EJBCA PKI structure for my TAK (tak.gov) server. I was able to sucessfully configure my CDP but I am stuck at the certificate enrollment part. I know there's a way to do it for Windows in the enterprise version however I have the community version and I need to configure it for a Linux server (non-Windows enrollment).
Is there anyone who could help me out?

**Comment by primetomas** [comment]
From Linux servers there are tons of different ways to enroll. Depending on what you want to enroll. These is nothing like MS auto-enrollment for Linux servers. But for web servers, you have ACME and certBot, which comes with all Linux's. For custom clients you can use simple scripts using curl (to the REST API or to the standard EST API).

So it's all depending on what you want you do.

Here are a few resources to get you started.
https://doc.primekey.com/ejbca/ejbca-operations/ejbca-ca-concept-guide/protocols
https://doc.primekey.com/ejbca/ejbca-integration/integrating-with-third-party-applications/using-certbot-to-issue-certificates-with-acme-to-an-apache-web-server
